### DISCUSSION TITLE
What is the workflow to update api/model/rpc files

### DISCUSSION THREAD

**Discussion by qazwsxedckll** [zeromicro/go-zero#2173] [Q&A]
For example, I wrote the .api file, created files with goctl and worte a lot of logics. What are the steps if I want to change the .api file.

How about files generated by goctl model. I wrote some code in xxxmodel_gen.go and I want to change the table structure now.

Is there something like flask-migrate in python?

update: It seems that I should add methods in the interface of xxxmodel.go instead of xxxmodel_gen.go.

**Comment by zhengjc6** [comment]
I just confronted the same problem as yours. I defined interfaces in xxxmodel_gen.go and wrote implements in another file named xxxmodel_cus.go  with the same package name.
